src/utils.jl

```diff
@@ -57,13 +57,60 @@ function insorted(item, collection)
     @inbounds return (index <= length(collection) && collection[index] == item)
 end
 
+
 """
-    generate_min_set(g, gen_func, Reps)
+    generate_min_set(g, gen_func, Reps; parallel=:threads)
```

Comment:
Why is `Reps` capitalized? AFAIK we don't do that anywhere else for variables greater than 1 or 2 chars.

Reply:
Also, with the new `Parallel` module, this should probably go into `Parallel/utils.jl`.

Reply:
I've included it in the parallel module.
Should I replace the implementation using `mapreduce` from the sequential model?